lib/airbrake/rake/tasks.rb

```diff
@@ -40,6 +41,15 @@
 
 The test exception was sent. Find it here: #{response['url']}
 OUTPUT
+    else
```

Comment:
So response will not be present **only** when you ignore env from which you're running the task?